### ISSUE TITLE
"Sort by" and "Show" options should not cause a change of context.

### ISSUE THREAD

**Issue by lschiff** [eScholarship/jschol#967]
Selecting a sort or display option should not cause a change of context. If a page reload is necessary, focus should be set to where the user left off rather than returning to the top of the page, for these kinds of series pages (Select (#c-sort1 and #c-sort2):
[https://escholarship.org/uc/bling_pubs/search?sort=asc&rows=10&start=0](https://escholarship.org/uc/bling_pubs/search?sort=asc&rows=10&start=0)

While the drop downs visually are at the top of the page, they are actually 21 and 22 in the tabbing order. It is true that if the “Skip to main content” link is used, the user will jump to 19 so only need to tab forward four or five times to reach the drop-downs, but this still takes them out of the context of where they were in the page when they made the selection and according to 3.2.2 there shouldn’t be a change of context on input. This could be especially disorienting for users who can’t see the page and might not understand why it has reloaded or if their selection has been saved.

This is [accessibility item 75](https://docs.google.com/spreadsheets/d/1kIMiW09tBtk5d3kJjzUMSCo1tEWPn7bUc0BYAJes9Hw/edit?gid=0#gid=0)

**Comment by nicktahani** [comment]
@justingonder please review on stg.

**Comment by justingonder** [comment]
@nicktahani confirming this looks good (visually) to me. Noting that the attached bug #748 hasn't' been addressed, but we have a separate issue for that.
